app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -81,11 +83,12 @@ class ScanSimpleIdScreen(context: Context, attrs: AttributeSet) : ConstraintLayo
   }
 
   fun showShortCodeValidationError() {
-    TODO("not implemented")
+    shortCodeErrorText.visibility = View.VISIBLE
+    shortCodeErrorText.text = resources.getString(R.string.scansimpleid_shortcode_error)
```

Comment:
Since the error text is static, we can set it as a resource in the layout and just hide/show it here.

Reply:
This will change once we implement another error message for empty short code.
Let's keep it this way for now.